ci_tutorial/products_pipeline.rst

```diff
@@ -0,0 +1,103 @@
+Products pipeline
+==================
+
+The **products pipeline** responds a more challenging question: does my "products" build correctly with the latest changes and new versions that have been done
```

Comment:
```suggestion
The **products pipeline** responds to a more challenging question: does my "products" build correctly with the latest changes and new versions that have been done
```

Reply:
in general a few grammar / wording choice things in this paragraph. consider taking a second look

including "tested against the organization's important products" (note the missing `'s`)